src/FormCheck.js

```diff
@@ -79,6 +79,9 @@ const propTypes = {
   /** Manually style the input as invalid */
   isInvalid: PropTypes.bool.isRequired,
 
+  /** Display feedback as a tooltip. */
+  tooltip: PropTypes.bool,
```

Comment:
**suggestion:** maybe this should be something like `feedbackTooltip`? it'd be more explicit given context

Reply:
Sounds good. I changed this in both FormCheck and FormFile